platform/admin/core/admin-core-insecuredefaults/src/test/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollectorTest.java

```diff
@@ -64,6 +69,30 @@ public void validate() {
     assertThat(details.iterator().next(), is(BASE_ALERT_TEST_MESSAGE + 0));
   }
 
+  @Test
+  public void validateUserPropertiesNotice() throws IOException {
+    // Setup
+    final int validatorCount = 3;
+    InsecureDefaultsCollector bean = createInsecureDefaultsServiceBean(validatorCount);
+    ArgumentCaptor<Event> captor = ArgumentCaptor.forClass(Event.class);
+
+    UsersPropertiesFileValidator validator = new UsersPropertiesFileValidator();
+    validator.setUsersPropertiesDeletion(true);
+
+    assertTrue(USERS_PROPERTIES_FILE_PATH.getParent().toFile().mkdirs());
+    assertTrue(USERS_PROPERTIES_FILE_PATH.toFile().createNewFile());
```

Comment:
🖊   I believe our convention is to use `assertThat` with a supplied failure reason.